Shows build and link errors in configure for ease of debugging which
options require what.
diff --git a/scripts/checks.sh b/scripts/checks.sh
index 64cbbf3..fab4d9b 100644
--- a/scripts/checks.sh
+++ b/scripts/checks.sh
@@ -425,7 +425,7 @@ try_compile()
 		echo "$1" > $__src || exit 1
 		shift
 		__cmd="$CC -c $CFLAGS $@ $__src -o $__obj"
-		$CC -c $CFLAGS "$@" $__src -o $__obj 2>/dev/null
+		$CC -c $CFLAGS "$@" $__src -o $__obj
 		;;
 	cxx)
 		__src=`tmp_file prog.cc`
@@ -433,7 +433,7 @@ try_compile()
 		echo "$1" > $__src || exit 1
 		shift
 		__cmd="$CXX -c $CXXFLAGS $@ $__src -o $__obj"
-		$CXX -c $CXXFLAGS "$@" $__src -o $__obj 2>/dev/null
+		$CXX -c $CXXFLAGS "$@" $__src -o $__obj
 		;;
 	esac
 	return $?
@@ -451,7 +451,7 @@ try_compile_link()
 		echo "$1" > $__src || exit 1
 		shift
 		__cmd="$CC $__src -o $__exe $CFLAGS $LDFLAGS $@"
-		$CC $__src -o $__exe $CFLAGS $LDFLAGS "$@" 2>/dev/null
+		$CC $__src -o $__exe $CFLAGS $LDFLAGS "$@"
 		;;
 	cxx)
 		__src=`tmp_file prog.cc`
@@ -459,7 +459,7 @@ try_compile_link()
 		echo "$1" > $__src || exit 1
 		shift
 		__cmd="$CXX $__src -o $__exe $CXXFLAGS $CXXLDFLAGS $@"
-		$CXX $__src -o $__exe $CXXFLAGS $CXXLDFLAGS "$@" 2>/dev/null
+		$CXX $__src -o $__exe $CXXFLAGS $CXXLDFLAGS "$@"
 		;;
 	esac
 	return $?
